docs/options.md

```diff
@@ -1,5 +1,21 @@
 # Options
 
+## Configuration
+
+Options for Conftest can be configured on the command line as flag arguments, or as environment variables that are prefixed with `CONFTEST_`.
+
+Conftest also supports a simple configuration file which can be used to store configuration settings for the `conftest` command.
+
+Create a `conftest.toml` configuration file like the following:
```

Comment:
Worth noting where this is discovered from.

Reply:
👍  agreed.